llvm/lib/Target/AMDGPU/SIISelLowering.cpp

```diff
@@ -12603,20 +12627,21 @@ bool SITargetLowering::isCanonicalized(SelectionDAG &DAG, SDValue Op,
     case Intrinsic::amdgcn_trig_preop:
     case Intrinsic::amdgcn_log:
     case Intrinsic::amdgcn_exp2:
+    case Intrinsic::amdgcn_sqrt:
```

Comment:
unrelated bonus change?

Reply:
Not unrelated, this is one of the things that came up in LLVM's testsuite where previously we were able to optimize away a `fcanonicalize` because we saw `sqrt`, but now we only check later and see `amdgcn_sqrt`.